scanner/src/main/kotlin/Scanner.kt

```diff
@@ -648,7 +648,7 @@ class Scanner(
             }
         }
 
-        logger.info { "Created file list for ${provenancesMissingFileLists.size} provenances in $duration." }
+        logger.info { "Created file lists for ${provenancesMissingFileLists.size} provenances in $duration." }
```

Comment:
Commit message nit: The type could be "docs" as this is only a log statement, i.e. not a logic fix, and the word "fix" is already part of the actual title.